src/components/Notifications/EventLog/usePrimaryToolbarFilterConfigWrapper.tsx

```diff
@@ -0,0 +1,180 @@
+
+                            if (activeFilter.chips.length === 0) {
+                                idxToRemove.push(idx);
+                            }
+
+                            return true;
+                        }
+
+                        return false;
+                    });
+                });
+
+                idxToRemove.forEach((indexRemove, idx) => {
+                    prev.splice(indexRemove - idx, 1);
+                });
+            }));
+        }
+    }, [ defaultDelete, setCustomFilters ]);
+
+    const mapToEventLogCustomFilter = React.useCallback((
+        filters: EventLogFilters,
+        bundles?: readonly Schemas.Facet[]
+    ) => {
+        const eventLogCustomFilters = (filters.bundle as string[])?.map(filterBundle => {
+            const indexOfBundle = bundles?.findIndex(bundle => bundle.name === filterBundle);
+            const bundle = indexOfBundle !== -1 && indexOfBundle !== undefined ? (bundles as Schemas.Facet[])[indexOfBundle] : undefined;
+
+            const bundleDisplayName = bundle?.displayName;
+            const applicationChips = (!bundle?.children ?
+                [
+                    {
+                        name: 'Loading',
+                        value: filterBundle,
+                        isRead: true
+                    }
+                ]
+                :
+                (bundle.children.length !== 0 ?
+                    bundle?.children?.map(application => ({
+                        name: application.displayName,
+                        value: application.name,
+                        isRead: true
+                    }))
+                    :
+                    [
+                        {
+                            name: bundle.displayName,
+                            value: bundle.name,
+                            isRead: true
+                        }
+                    ]
+                )
+            );
+
+            return {
+                bundleId: filterBundle,
+                category: bundleDisplayName || `${filterBundle} Loading...`,
+                chips: (filters.application as string[])?.map(filterApplication => {
+                    const appDisplayName = bundle?.children?.find(application => application.name === filterApplication)?.displayName;
+                    return {
+                        name: appDisplayName as string,
+                        value: filterApplication,
+                        isRead: true
+                    };
+                }) || applicationChips
+            };
+        }) || [];
+
+        return eventLogCustomFilters;
+    }, []);
+
+    React.useEffect(() => {
+        setCustomFilters(mapToEventLogCustomFilter(filters, bundles.length !== 0 ? bundles : undefined));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [ bundles, mapToEventLogCustomFilter ]);
+
+    toolbarConfig.filterConfig.items[1] = useMemo(() => {
```

Comment:
It's always on the position `1` ?

Reply:
It can technically be at any position, I just have it always at 1

Reply:
I have a ticket that adds some filters for EventLog ([NOTIF-324](https://issues.redhat.com/browse/NOTIF-324) I think), I will make this dynamic under that PR... resolving for now.